### ISSUE TITLE
Serverless: Configuration warning at 'functions['export-latest'].events[0].http': unrecognized property 'throttling'

### ISSUE THREAD

**Issue by crushjelly** [DianaIonita/serverless-api-gateway-throttling#9]
I am getting this warning `Serverless: Configuration warning at 'functions['export-latest'].events[0].http': unrecognized property 'throttling'` when using http event. 
```

    events:
      - http:
          path: export/item
          method: get
          private: true
          throttling:
            maxRequestsPerSecond: 2000
            maxConcurrentRequests: 1000
```

When I use global _apiGatewayThrottling_ configuration it works fine.
Is my configuration bad?

**Comment by DianaIonita** [comment]
Hi @crushjelly,

Thanks for raising the issue. Your configuration looks fine. Recently, the Serverless framework added warnings whenever it encounters configuration that isn't native, such as the kind that this plugin uses.

Plugin developers have the option of [extending the validation schema](https://www.serverless.com/framework/docs/providers/aws/guide/plugins#extending-validation-schema), which I plan to do soon, and it will make the warning go away.

In the meantime, you have the option of [turning off validation](https://www.serverless.com/framework/docs/configuration-validation/) to remove the warning.

**Comment by DianaIonita** [comment]
Released [v1.0.2](https://github.com/DianaIonita/serverless-api-gateway-throttling/releases/tag/1.0.2).
